### DISCUSSION TITLE
Branch.io Deep Linking and Smart App Banner

### DISCUSSION THREAD

**Discussion by jsaristi** [inspireui/support#5572] [Working on Features]
I would like them to add vital functionality to this app that would make the app supremely relevant. the functionality would work in plugin addon of "woocommerce mstore api" that allows you to place a banner on the main page that invites users to download the application. as well as to redirect them to the page that the user is currently on and upload it to the page.

![Layer-1](https://user-images.githubusercontent.com/29686082/180206029-ddd659e8-7ae7-4970-bd3e-66884ca5694b.png)

I know that there is the functionality of Firebase dynamic linking but it only works to create links to a specific product in the application but this has to be done manually and also does not allow placing a banner on the web page to open it automatically from the app the perfect functionality that This is done by Branch (https://branch.io/) which generates the smart banner on the web that not only invites you to download the app, but when it is already downloaded, it shows the message to open this product in the application.

<img width="1078" alt="Captura de Pantalla 2022-07-21 a la(s) 6 49 53 a m" src="https://user-images.githubusercontent.com/29686082/180207390-de8bb3ab-ccbd-4513-a63c-3b55dd6ccf3c.png">

https://branch.io/what-is-deep-linking/

It also makes the sales (ROI) improve which makes the app much more impactful and this app much more valuable to the users.

![urlgenius-walmart-smart-banner-screenshot250x250-250x215](https://user-images.githubusercontent.com/29686082/180208841-af5bc778-06a3-4719-b1ca-9a356f82ae8e.png)

<img width="1269" alt="Captura de Pantalla 2022-07-21 a la(s) 6 50 13 a m" src="https://user-images.githubusercontent.com/29686082/180207360-3d87b897-432f-4d0c-8c8d-f4d639d58eb3.png">

<img width="1316" alt="Captura de Pantalla 2022-07-21 a la(s) 6 50 59 a m" src="https://user-images.githubusercontent.com/29686082/180207489-8e841892-e99f-463f-875d-36ff49d1f7a0.png">

**Comment by jsaristi** [comment]
Please pay attention and vote for this very important feature.

**Comment by simonpham** [comment]
https://github.com/inspireui/support/discussions/5539

**Comment by jsaristi** [comment]
Thank you very much for working on this I look forward to this new feature

**Reply by HanielQC** [reply]
Hi, we have released Smart Engagement Banner, refer to our Video https://youtu.be/oFFnE1Ssm74 for more detail. Thank you!

**Reply by jsaristi** [reply]
Hello, thanks for answering.

I think you did not understand me since the functionality I was referring to is totally different from the one you developed.

I was referring to a banner that goes inside the web page that redirects customers from the product they are viewing to the product page inside the application as shown in branch.io that works with a deep link.

Below I leave an image of what I was referring to and a link to a page that more or less uses this.

![appfollow_299bb3f3-a154-4df4-8847-0b4056231a40](https://user-images.githubusercontent.com/29686082/200995003-3dceeefb-0bdf-4c04-98d7-95589986c328.jpg)

[Redfin.com](https://www.redfin.com/)

See how the button redirects me to the app and shows me the same property of the webpage in the app.

![IMG_0141](https://user-images.githubusercontent.com/29686082/200995324-878dd13e-28ea-4f79-b231-198ab3901f37.PNG)

the property in the app

![IMG_0142](https://user-images.githubusercontent.com/29686082/200995141-cb73fb42-d7ce-478d-9bd3-978cfa8a1921.PNG)

Thats what i mean in the feature to offers branch.io

<img width="1135" alt="Captura de pantalla 2022-11-09 a la(s) 10 45 54 p m" src="https://user-images.githubusercontent.com/29686082/200995952-64dacd8d-a7c5-486a-8e5d-55de881285e0.png">

https://branch.io/what-is-deep-linking/

**Comment by tryabp** [comment]
thats the feature i was looking for! i hope @fluxstore team will look into it.
